stdlib/jms/src/main/ballerina/jms/queue_receiver.bal

```diff
@@ -31,17 +31,17 @@ public type QueueReceiver object {
     public function init(QueueReceiverEndpointConfiguration c) {
         self.config = c;
         self.consumerActions.queueReceiver = self;
-        match (c.session) {
-            Session s => {
-                match (c.queueName) {
-                    string queueName => {
-                        self.createQueueReceiver(s, c.messageSelector);
-                        log:printInfo("Message receiver created for queue " + queueName);
-                    }
-                    () => {}
-                }
+        var session = c.session;
+        if (session is Session) {
+            var queueName = c.queueName;
+            if (queueName is string) {
+                self.createQueueReceiver(session, c.messageSelector);
+                log:printInfo("Message receiver created for queue " + queueName);
+            } else {
+                log:printInfo("Message receiver is not properly initialised for queue");
```

Comment:
In some places, `initialised` is used and in some other places `initialized` is used. Better to change `initialised` -> `initialized` in all applicable places since we use American English.

Reply:
Resolved